core/src/main/resources/com/graphhopper/custom_models/hike.json

```diff
@@ -1,5 +1,4 @@
 // to use this custom model you set the following option in the config.yml:
-// graph.vehicles: roads,foot
 // graph.encoded_values: hike_rating,average_slope,max_slope
```

Comment:
`graph.encoded_values` should not be necessary? Or did you leave it there for path details?